docs/reference/developer/notebook-style-guide.ipynb

```diff
@@ -17,7 +17,8 @@
     "\n",
     "### Saving notebooks\n",
     "\n",
-    "- Avoid storing outputs before saving or committing a notebook.\n",
+    "- Notebooks that are part of documentation get executed by Sphinx during the documentation build process.\n",
+    "  For such notebooks, avoid saving outputs before committing a notebook.\n",
```

Comment:
Do we ever want to commit outputs to git? I'd hate to grow the repo by megabytes because of it.